modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -1025,14 +1110,14 @@ events
 [discrete]
 == `console.topology/adapter/build`
 
-`BuildAdapter` contributes an adapter to adapt element to data that can be used by the `Build` component.
+`BuildAdapter` contributes an adapter to adapt element to data that can be used by the Build component.
 
 [cols=",,,",options="header",]
 |===
 |Name |Value Type |Optional |Description
 |`adapt`
 |`CodeRef<(element: GraphElement) => AdapterDataType<BuildConfigData> | undefined>`
-|no |
+|no |adapter to adapt element to data that can be used by Build component.
```

Comment:
formatting issue in the table
![Screenshot 2023-03-30 at 2 12 58 PM](https://user-images.githubusercontent.com/12692381/228745716-4a4e5d38-a242-46d6-b86f-c99dd13c6241.png)